### PR TITLE
chore: Add usage of cross-env-shell

### PR DESCRIPTION
**Pull request by ciband**
## Proposed changes
Added use of cross-env-shell to add Windows support to the pre commit checkin script.

## Types of changes

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [X] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
<!--

## Further comments
This is simple and works, but is limited in that the single, cross-platform script can only use syntax and commands that work in both bash (*nix) and batch (Windows).

I am open to other ideas if we wanted to use two scripts instead.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ jobs:
       - save_cache:
           key: 'core-node10-{{ checksum "checksum.txt" }}-1'
           paths:
+            - ./packages/client/node_modules
```
Comment (faustbrian) [inline .circleci/config.yml]: This package no longer exists in this repository, please clean up those old files or do a fresh clone.

Comment (ciband) [reply .circleci/config.yml]: Will do.  I'll get this done tonight.

Comment (ciband) [reply .circleci/config.yml]: @faustbrian  done

**Comment by faustbrian** [comment]
@ciband conflicts.

**Comment by ciband** [comment]
@faustbrian done

**Pull request merged**